### ISSUE TITLE
Unable to save log file on s3 while training

### ISSUE THREAD

**Issue by Aditi00a** [JohnSnowLabs/spark-nlp#5899]
## Description
saving the model logs in the ner pipeline using .setOutputLogsPath( "./Path_to_s3")
however it gives this issue -  Wrong FS: s3:// , expected: hdfs://'

## Expected Behavior
the logs should have been saved in the s3 directory mentioned

## Current Behavior
 it gives this error-  Wrong FS: s3:// , expected: hdfs://'

## Steps to Reproduce
1. run the bert model piepline and save the log files on S3

## Your Environment

* Spark NLP version `sparknlp.version()`: 3.1.1
* Apache NLP version `spark.version`: 3.1.2
* Java version `java -version`:
* Setup and installation (Pypi, Conda, Maven, etc.):
* Operating System and version:
* Link to your project (if any):

**Comment by maziyarpanahi** [comment]
Hi,

Could you please complete the template with a code and full error? (I am not sure where you are, what you are trying to do, and a full error stack would help to see if S3 is not supported or something else went wrong)

**Comment by Aditi00a** [comment]
i am running a sparknlp training pipeline job on sagemaker and want to save the output log file to s3

**Comment by maziyarpanahi** [comment]
Thanks, we still need the rest of the missing info in the template, the versions, the steps, everything that helps us to reproduce this issue.

**Comment by Aditi00a** [comment]
running this code:
    args = parser.parse_args()

    spark = sparknlp.start()
    print("Spark NLP version: ", sparknlp.version())
    print("Apache Spark version: ", spark.version)

    training_file = "s3://" + os.path.join(args.s3_input_bucket,
                                           args.s3_input_key_prefix,
                                           args.training_file)

    testing_file = "s3://" + os.path.join(args.s3_input_bucket,
                                          args.s3_input_key_prefix,
                                          args.testing_file)

    log_file = "s3a://" + os.path.join(args.s3_output_bucket,
                                     args.s3_output_key_prefix,
                                     "model_logs")

    training_data = CoNLL().readDataset(spark, training_file)
    training_data.show(10)

    # we use BERT Tiny
    bert_annotator = BertEmbeddings.pretrained('small_bert_L2_128', 'en'). \
        setInputCols(["sentence", 'token']). \
        setOutputCol("bert"). \
        setBatchSize(8)

    test_data = CoNLL().readDataset(spark, testing_file)
    test_data = bert_annotator.transform(test_data)

    test_data_parquet = "test.parquet"
    test_data.write.parquet(test_data_parquet)

    nerTagger = NerDLApproach(). \
        setInputCols(["sentence", "token", "bert"]). \
        setLabelColumn("label"). \
        setOutputCol("ner"). \
        setMaxEpochs(5). \
        setLr(0.001). \
        setPo(0.005). \
        setBatchSize(32). \
        setEvaluationLogExtended(True). \
        setEnableOutputLogs(True). \
        setOutputLogsPath(log_file). \
        setTestDataset(test_data_parquet)

    pipeline = Pipeline(stages=[bert_annotator, nerTagger])

    ner_model = pipeline.fit(training_data)

    predictions = ner_model.transform(
        test_data.select("sentence", "token", "label"))

    eval_data = predictions.select(F.explode(
        F.arrays_zip('token.result', 'label.result', 'ner.result')).alias(
        "cols")) \
        .select(F.expr("cols['0']").alias("token"),
                F.expr("cols['1']").alias("ground_truth"),
                F.expr("cols['2']").alias("prediction"))

**configuration -**
configuration = [
{
"Classification": "spark-defaults",
"Properties": {
"spark.yarn.stagingDir": "hdfs:///tmp",
"spark.yarn.preserve.staging.files": "true",
"spark.kryoserializer.buffer.max": "2000M",
"spark.serializer": "org.apache.spark.serializer.KryoSerializer",
"spark.driver.maxResultSize": "0",
"spark.jars.packages":
"com.johnsnowlabs.nlp:spark-nlp-spark24_2.11:3.1.3",
"spark.driver.memory": "12g",
"spark.driver.cores": "3",
"spark.executor.memory": "12g",
"spark.executor.cores": "4",
"spark.driver.memoryOverhead": "3g"
}
}
]
**Error while running code**

![image](https://user-images.githubusercontent.com/88418260/128175502-1231ab3a-1140-47e0-9d69-f9221aa9f33d.png)

**Comment by maziyarpanahi** [comment]
This is true, the S3 fileSystem is not supported for the logging. It doesn't support append. (must be local, HDFS, or DBFS.)
